roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```
@@ -110,9 +110,8 @@
     ignore_errors: true
     changed_when: true
   - name: Detect ovirt-hosted-engine-ha version
-    ansible.builtin.command: >-
-      {{ ansible_python.executable }} -c
-      'from ovirt_hosted_engine_ha.agent import constants as agentconst; print(agentconst.PACKAGE_VERSION)'
+    ansible.builtin.shell: >-
+      rpm -qi ovirt-hosted-engine-ha | grep "Version" | awk '{print $3}'
```

maybe you can try this one:
```suggestion
      rpm -q --queryformat '%{VERSION}' ovirt-hosted-engine-ha
```

---

Thanks, CI fails with the following error:
`command-instead-of-module rpm used in place of yum or rpm_key module` 
that comes from ansible-lint, so I'll try with the `yum` module.